### DISCUSSION TITLE
3.5 mm to 2.5 mm adapters

### DISCUSSION THREAD

**Discussion by Jens5395** [emporia-vue-local/esphome#6] [Q&A]
I placed another order with Emporia which includes some flexible sensors and some thin clamp sensors. Both of these are 200 amp sensors and I suspect they both come with 3.5 mm plugs. I will need to be able to hook the high power sensors to the 2.5 mm inputs so I am looking for an adapter. The adapters that are available from Emporia are described as allowing gen 1 devices to plug into the 2.5 mm sockets of the Vue 2. There is no mention of using this adapter to connect a high current vue 2 sensor to connect to the 2.5 mm ports of the Vue 2.
Also, looking at the adapters that Emporia sells, the male 2.5 mm  end is a three pole (stereo) plug. The 50A Vue 2 sensors use a two pole (mono) plug.
My question: has anyone had an opportunity to connect a high power sensor to the 2.5 mm ports?
Will an adapter going from a female 3.5 mm stereo socket to a male 2.5 mm male stereo plug work? I was unable to find an adapter in a two pole (mono) version but they are readily available if a stereo plug will work in this situation.

BTW, I ordered the rope sensors because my main power panel wires are too fat for the regular 200A sensors and I am also quite tight for room. I will likely end up sensing the main power by wrapping the rope sensors around the main bus bars. Also, the regular 200A sensors are just a tiny bit too small to fit around the aluminum wires of my 100A electrical central heater. This is where the thin clamps come into play and they will have to be attached to a 2.5 mm port.

**Comment by flaviut** [comment]
I've never heard of anyone doing this, but I don't see why it wouldn't work.

You'll probably need to [to use filters](https://esphome.io/components/sensor/index.html) to calibrate the data before it is useful, since the flexible clamps likely have a different ratio from the standard clamps.
